src/glue/security-configuration.ts

```diff
@@ -103,18 +103,48 @@ export interface S3Encryption {
  * Configuration for the Glue SecurityConfiguration resource.
  */
 export interface SecurityConfigurationProps extends ResourceProps {
+  /**
+   * Cloudwatch Encryption Settings
+   * 
+   * @see [AWS::Glue::SecurityConfiguration EncryptionConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-securityconfiguration-encryptionconfiguration.html#cfn-glue-securityconfiguration-encryptionconfiguration-cloudwatchencryption)
+   */
   readonly cloudWatchEncryption?: CloudWatchEncryption;
+  /**
+   * The encryption configuration for job bookmarks.
+   * 
+   * @see [AWS::Glue::SecurityConfiguration EncryptionConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-securityconfiguration-encryptionconfiguration.html#cfn-glue-securityconfiguration-encryptionconfiguration-jobbookmarksencryption)
+   */
   readonly jobBookmarksEncryption?: JobBookmarksEncryption;
+  /**
+   * Name for the Security Configuration.
+   */
   readonly name?: string;
+  /**
+   * The encyption configuration for Amazon Simple Storage Service (Amazon S3) data.
+   * 
+   * @see [AWS::Glue::SecurityConfiguration EncryptionConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-securityconfiguration-encryptionconfiguration.html#cfn-glue-securityconfiguration-encryptionconfiguration-s3encryptions)
+   */
   readonly s3Encryption?: S3Encryption;
 }
 
 export class SecurityConfiguration extends Resource {
   // Input properties
-  public readonly cloudWatchEncryption?: CloudWatchEncryption;
-  public readonly jobBookmarksEncryption?: JobBookmarksEncryption;
-  public readonly name?: string;
-  public readonly s3Encryption?: S3Encryption;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now